src/jit/codegencommon.cpp

```diff
@@ -3661,12 +3659,18 @@ void CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg, bool* pXtraRegClobbere
             regNumber regNum = genMapRegArgNumToRegNum(regArgNum + i, regType);
 
 #if !defined(UNIX_AMD64_ABI)
-            // lvArgReg could be INT or FLOAT reg. So the following assertion doesn't hold.
-            // The type of the register depends on the classification of the first eightbyte
-            // of the struct. For information on classification refer to the System V x86_64 ABI at:
-            // http://www.x86-64.org/documentation/abi.pdf
+// lvArgReg could be INT or FLOAT reg. So the following assertion doesn't hold.
+// The type of the register depends on the classification of the first eightbyte
+// of the struct. For information on classification refer to the System V x86_64 ABI at:
+// http://www.x86-64.org/documentation/abi.pdf
```

Comment:
This comment also needs to be updated, since the code below doesn't apply only to x64/ux (and I'd suggest getting rid of the System V references; they are unnecessary and confusing, IMO).